src/components/document/student-group-view.tsx

```diff
@@ -43,13 +43,24 @@ export const StudentGroupView:React.FC<IProps> = ({ groupId, setGroupId }) => {
   const [focusedGroupUser, setFocusedGroupUser] = useState<GroupUserModelType | undefined>();
   const selectedGroupId = groupId || (groups.allGroups.length ? groups.allGroups[0].id : "");
   const groupUsers = getGroupUsers(user.id, groups, documents, selectedGroupId);
-  const group = groups.getGroupById(selectedGroupId);
-
   const ui = useUIStore();
+  const group = groups.getGroupById(selectedGroupId);
   const isStudentViewActiveTab = (ui.activeNavTab === "student-work");
   const isChatPanelShown = ui.showChatPanel;
   const shrinkStudentView  = isStudentViewActiveTab && isChatPanelShown;
-  const classes = classNames("document", "student-group-view", {"shrink-student-view": shrinkStudentView});
+  const documentSelectedForComment = ui.showChatPanel && ui.selectedTileIds.length === 0;
+  const studentGroupViewClasses = classNames( "editable-document-content", "document", "student-group-view",
+  {"shrink-student-view": shrinkStudentView}, {"comment-select" : documentSelectedForComment});
+
+  if (focusedGroupUser && isStudentViewActiveTab){
+    const id = focusedGroupUser.id;
+    const foundUser = groupUsers.find(obj => obj.user.id === id);
+    if (foundUser && foundUser.doc && foundUser.doc.key && foundUser.doc.groupId){
+      const subTab = foundUser.doc.groupId; //G1, G2, etc
```

Comment:
May be useful to use a more descriptive var name since we use subTab for the react-tab subTabs (`groupSubTab`?) and/or write a comment that we are using the group buttons as a psuedo subtabs.

Reply:
Yeah that makes sense, I'll use that variable name instead.